docs/conf.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""Sphinx configuration file."""
+
+import os
+import time
+
+import sphinx_rtd_theme
+html_theme = 'sphinx_rtd_theme'
+html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+
+# html_theme = "alabaster"
+html_logo = "logo.png"
+
+
+project = "maplab"
+master_doc = 'index'
+author = "ASL Mapping Team"
+copyright = "{}, {}".format(time.strftime("%Y"), author)
+html_last_updated_fmt = "%c"
+pygments_style = "sphinx"
+templates_path = ["_templates"]
+
+# Check https://holzhaus.github.io/sphinx-multiversion/master/configuration.html#tag-branch-remote-whitelists
+smv_tag_whitelist = r'^.*$'                   # Include all tags
+smv_branch_whitelist = r'^.*$'                # Include all branches
+smv_remote_whitelist = r'^.*$'                # Use branches from all remotes
+
+extensions = [
+     'breathe', 'exhale', 'sphinx.ext.autosectionlabel', 'recommonmark', 'sphinx_multiversion',
+]
+
+html_theme_options = {
+    'canonical_url': 'maplab.asl.ethz.ch',
+    'analytics_id': 'UA-XXXXXXX-1',  #  Provided by Google in your dashboard
+    'logo_only': True,
+    'display_version': True,
+    'prev_next_buttons_location': 'bottom',
+    'style_external_links': True,
+    'style_nav_header_background': 'green',
+    # Toc options
+    'collapse_navigation': True,
+    'sticky_navigation': True,
+    'navigation_depth': 4,
+    # 'includehidden': True,
+    # 'titles_only': False
+}
+
+html_context = {
+    'display_github': True,
+    'github_repo': 'maplab',
+    'github_user': 'ethz-asl',
+    'github_version': 'develop-docs',
```

Comment:
What is this parameter and does it matter? It seems to be the current branch name, do we need to change this or something when you branch out?

Reply:
Yes, this has to do with the `Edit on GitHub` if I remember correctly...Should point to the correct branch.